website/route-lockfile.txt

```diff
@@ -12,7 +12,6 @@
 /ar/cookbook/cosmos/
 /ar/cookbook/grafting/
 /ar/cookbook/near/
-/ar/cookbook/quick-start/
```

Comment:
@benface this concerns me a bit. Do you know what's going on here?

Reply:
Responded in Slack